common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -250,10 +268,17 @@ define('WordCloudMain', [], function () {
                 .attr('transform', 'translate(' + (0.5 * this.width) + ',' + (0.5 * this.height) + ')')
                 .selectAll('text')
                 .data(words)
-                .enter().append('g');
+                .enter()
+                .append('g')
+                .attr('aria-labelledby', function(d) {
```

Comment:
They should be unique though, since I'm using the word cloud ID in the individual ID.